<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trainerQuery</class>
 <widget class="QWidget" name="trainerQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>教练信息查询</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="queryGroupBox">
     <property name="title">
      <string>查询条件</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QComboBox" name="conditionComboBox">
        <item>
         <property name="text">
          <string>所有教练</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>教练ID</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>教练姓名</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>联系电话</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>邮箱</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="valueLineEdit"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="queryButton">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QGroupBox" name="modifyGroupBox">
     <property name="title">
      <string>修改教练信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="specialtyLabel">
        <property name="text">
         <string>擅长训练项目:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="specialtyLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="workScheduleLabel">
        <property name="text">
         <string>工作排班信息:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextEdit" name="workScheduleTextEdit"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>保存修改</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>